discord/enums.py

```diff
@@ -452,8 +452,10 @@ def target_type(self) -> Optional[str]:
             return 'thread'
         elif v < 122:
             return 'integration_or_app_command'
-        elif v < 144:
+        elif v < 143:
             return 'auto_moderation'
+        elif v == 143:
+            return 'user'
```

Comment:
This feels a bit weird. What's the reason for that?

Reply:
To change the target type for automod_block_message actions from auto_moderation to user.